index.html

```diff
@@ -2,295 +2,316 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+    />
+    
```

Comment:
## Your PR Title :ticket: 
![Screenshot from 2022-09-02 02-45-09](https://user-images.githubusercontent.com/93039359/188018550-b7f0f3d7-ad42-4f07-a2ca-a8afc4d303ad.png)
**[Optional]**
- [ ] Your PR has a title that is not so descriptive enough. Because **Add all files and folder** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Portfolio: Desktop Version** for your PR title. :+1:

Reply:
Thank You very much. That is a significant fault from my side.  I am incredibly sorry for that.